### PR TITLE
kbfs: update to 4.4.1

### LINKED ISSUE #14390: kbfs is very outdated, pulling from archived repo
Due to this `keybase-desktop` isn't fully functional.

* package: 
 kbfs-2.11.0_1
 
The new repo is https://github.com/keybase/client/tree/master/go/kbfs, previously reported on  #10353

### PR DESCRIPTION
**Pull request by Itamiko**
resolves #14390

### CODE REVIEW

File: srcpkgs/kbfs/template [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
 build_style=go
-go_import_path="github.com/keybase/kbfs"
-go_package="${go_import_path}/kbfsfuse ${go_import_path}/kbfsgit/git-remote-keybase ${go_import_path}/kbfstool ${go_import_path}/redirector"
-go_build_tags="production"
-depends="keybase util-linux"
+go_import_path="github.com/keybase/client/go/kbfs"
+go_package="${go_import_path}/kbfsfuse
+${go_import_path}/kbfsgit/git-remote-keybase
+${go_import_path}/kbfstool ${go_import_path}/redirector"
+makedepends="git"
```
Comment (Johnnynator) [inline srcpkgs/kbfs/template]: Should be `hostmakedepends`

File: srcpkgs/kbfs/template [hunk 2]
```diff
@@ -1,23 +1,28 @@
...
 license="BSD-3-Clause"
 homepage="https://keybase.io/docs/kbfs"
-distfiles="https://github.com/keybase/kbfs/archive/v${version}.tar.gz"
-checksum=0e97cab1c90592035ec6763e22c78c08a95dd3467419d15b9e45b3d0887b594d
+distfiles="https://github.com/keybase/client/archive/v${version}.tar.gz"
+checksum=dfac5ff4ecd5cd75f22ebece24c2e074cc83e38c7b207e074ca2c96ec3c96b7b
+
+do_configure() {
+	export GOPATH="${wrksrc}/.gopath"
+	vlicense LICENSE
```
Comment (Johnnynator) [inline srcpkgs/kbfs/template]: Why do you override `do_configure`? What is the issue with the one provided by the build_style? Also vlicense should stay in `post_install`

**Comment by Johnnynator** [comment]
@Vaelatern any opinion on maintainer change?

**Comment by Vaelatern** [comment]
I don't want this package to exist.

I want it to be part of the keybase client package. Working with that template is the correct course of action. We should not have multiple templates drawing from the same distfiles where it can be avoided.

**Comment by Itamiko** [comment]
kbfs should get its own package but as a sub-package for the client main instead of having kbfs use its own template.

same goes for all keybase-client related things. for example, kbfs and all other keybase-client submodules should be as a sub-package. or something along those lines.

**Comment by Piraty** [comment]
we have `kbfs-5.4.2_1` by now

**Pull request closed without merging**